modules/nf-core/limma/differential/main.nf

```diff
@@ -23,5 +25,10 @@ process LIMMA_DIFFERENTIAL {
     task.ext.when == null || task.ext.when
 
     script:
-    template 'limma_de.R'
+    if (type == 'rnaseq') {
+        template 'limma_de_rnaseq.R'
```

Comment:
How you envision "proper" integration? As single template? I will refactor that if needed.

Reply:
I mean, if the two scripts share enough logic, they should be one with a simple conditional. If the logic is quite divergent such that doing that adds too much complexity, these should be separate modules.

Reply:
Well I had similar doubts in the one hand this is still limma and differential abundance analysis and on the other site this one is combined with voom and there is lot of differences. So maybe we will create new module named limma/differential-voom? or simply limma/voom - what are your thoughts on the module naming?

Reply:
I think this should be limma/differential-voom. 

As in the comments below, I also think this should be, as much as possible, a thin wrapper around Limma functions only. Otherwise it's just your custom script, rather than a 'standard' Limma module.

Reply:
(as in other comments, I'm not sure we need edgeR here, so we'd only pin Limma)

Reply:
updating this older comment after newer ones below: I don't believe additional logic required for Voom merits its own module. We can add the Voom part in a conditional, and other changes (e.g. duplicateCorrelation) etc, apply equally to non-Voom Limma.